readthedocs/rtd_tests/tests/test_subprojects.py

```diff
@@ -143,6 +143,20 @@ def test_excludes_existing_subprojects(self):
             [''],
         )
 
+    def test_exclude_self_project_as_subproject(self):
+        user = fixture.get(User)
+        project = fixture.get(Project, users=[user])
+        subproject = project
+
+        form = ProjectRelationshipForm(
+            {'child': subproject.pk},
+            project=project,
+            user=user
+        )
+        self.assertNotIn(
```

Comment:
I think this won't be needed once you use `is_valid` method.

Reply:
This is needed for test the query set